client/src/app/containers/Game/ActionBox.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
```

Comment:
I think I've reviewed enough to have "reasonable" questions to ask. I'd love  to sync up this week about what is happening in TypeScript, some organizational goals, and some architectural questions. Designing client-side apps is still a relatively new thing for me, so I'm always open to learning from others.

From this point and below, I haven't _really_ reviewed things - this comment is acting as a personal marker/reminder.

Reply:
Sure. I'd love to sit down and answer your questions/find out what answers I _don't_ have (and there already are some). I think it's important to discuss and decide on architecture before it comes to the dreaded "yeah, we should have done this better from the start, but it's too painful to change now."